pl_bolts/datamodules/emnist_datamodule.py

```diff
@@ -0,0 +1,240 @@
+from typing import Any, Callable, Optional, Union
+
+from pl_bolts.datamodules.vision_datamodule import VisionDataModule
+from pl_bolts.datasets import EMNIST
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision import transforms as transform_lib
+else:  # pragma: no cover
+    warn_missing_pkg('torchvision')
+
+
+class EMNISTDataModule(VisionDataModule):
+    """
+    .. figure:: https://user-images.githubusercontent.com/4632336/123210742-4d6b3380-d477-11eb-80da-3e9a74a18a07.png
+        :width: 400
+        :alt: EMNIST
+
+    .. list-table:: Dataset information (source: `EMNIST: an extension of MNIST to handwritten
+        letters <https://arxiv.org/abs/1702.05373>`_ [Table-II])
+       :header-rows: 1
+
+       * - Split Name
+         - No. classes
+         - Train set size
+         - Test set size
+         - Validation set
+         - Total size
+       * - ``"byclass"``
+         - 62
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``byclass``
+         - 62
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``bymerge``
+         - 47
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``balanced``
+         - 47
+         - 112,800
+         - 18,800
+         - Yes
+         - 131,600
+       * - ``digits``
+         - 10
+         - 240,000
+         - 40,000
+         - Yes
+         - 280,000
+       * - ``letters``
+         - 37
+         - 88,800
+         - 14,800
+         - Yes
+         - 103,600
+       * - ``mnist``
+         - 10
+         - 60,000
+         - 10,000
+         - Yes
+         - 70,000
+
+    |
+
+    Standard EMNIST, train, val, test-splits and transforms
+
+    Transforms::
+
+        emnist_transforms = transform_lib.Compose([
+            transform_lib.ToTensor()
+        ])
```

Comment:
Could you update the docs here so that it matches with `default_transform`? Similar docs can be found at https://lightning-bolts.readthedocs.io/en/latest/datamodules_vision.html.

Reply:
@akihironitta Done!